docs/advanced-topics/wes/wes-agc-tutorial.rst

```diff
@@ -0,0 +1,329 @@
+                "workflow_url" : null
+              },
+              "state" : "COMPLETE",
+              "run_log" : null,
+              "task_logs" : [ {
+                "name" : "w.hello|e6ce6c0a-ae99-43de-accc-4e43183de73f",
+                "cmd" : [ "echo \"Hello from AGC\"" ],
+                "start_time" : "2022-03-04T17:19:52.341Z",
+                "end_time" : "2022-03-04T17:23:17.196Z",
+                "stdout" : "s3://agc-example123-us-west-2/project/dockstoreAgcTutorialProject/userid/userM2QLG/context/ctx1/cromwell-execution/w/8e8e9f4b-fb1a-41df-bc37-9396d6f97db5/call-hello/hello-stdout.log",
+                "stderr" : "s3://agc-example123-us-west-2/project/dockstoreAgcTutorialProject/userid/userM2QLG/context/ctx1/cromwell-execution/w/8e8e9f4b-fb1a-41df-bc37-9396d6f97db5/call-hello/hello-stderr.log",
+                "exit_code" : 0
+              } ],
+              "outputs" : {
+                "id" : "8e8e9f4b-fb1a-41df-bc37-9396d6f97db5",
+                "outputs" : {
+                  "w.hello.out" : "Hello from AGC"
+                }
+              }
+            }
+
+    Notice that the output for the worklow ``w``, task ``hello`` is "Hello from AGC".
+
+FastQ read counts workflow
+--------------------------
+The Dockstore entry associated with this workflow can be found here `agc-fastq-read-counts <https://dockstore.org/workflows/github.com/dockstore-testing/wes-testing/agc-fastq-read-counts:main?tab=info>`_.
+
+This WDL workflow tabulates read counts of the input fastq file.
+
+    *Dockstore.wdl*
+
+    .. code:: text
+
+        version 1.0
+
+        workflow fastqReadCounts {
+
+            call countFastqReads
+
+            output {
+                File totalReadsFile = countFastqReads.totalReadsFile
+            }
+        }
+
+
+
+        task countFastqReads {
+
+            input {
+                Array[File] inputFastq
+
+                Int memSizeGB = 4
+                Int diskSizeGB = 128
+                String dockerImage = "biocontainers/samtools:v1.9-4-deb_cv1"
+            }
+
+            command <<<
+
+                set -o pipefail
+                set -e
+                set -u
+                set -o xtrace
+
+                READ_COUNT=0
+
+                for fq in ~{sep=' ' inputFastq}
+                do
+                      FILE_COUNT=$(zcat "${fq}" | wc -l )/4
+                      READ_COUNT=$(( $READ_COUNT + $FILE_COUNT ))
+                done
+
+                echo $READ_COUNT > total_reads.txt
+            >>>
+
+            output {
+
+                File totalReadsFile  = "total_reads.txt"
+            }
+
+            runtime {
+                memory: memSizeGB + " GB"
+                disks: "local-disk " + diskSizeGB + " SSD"
+                docker: dockerImage
+                preemptible: 1
+            }
+        }
+
+1. This workflow takes an array of files as an input. Create a file named ``input.json`` in your working directory with contents:
```

Comment:
If it works :), you could take the opportunity to create the input.json scaffold using `dockstore convert`.

Reply:
Once AGC modifies the input JSON issue we are having, then we may be able to provision the test parameter files by default for the user. (i.e. use the test parameter file on the Dockstore entry if it's provided and not overwritten by the `--json` parameter). Created an issue: https://github.com/dockstore/dockstore/issues/4764